assets/app.js

```diff
@@ -69,13 +69,36 @@ $(function () {
         });
     });
 
+    let selectLangField = $('#lang');
```

Comment:
It looks like `$select2` is already this same selector. Also, note that the convention is to prefix jQuery objects with `$`.